PluginsManager/OfficialPluginsList.cs

```diff
@@ -51,19 +51,18 @@ internal static async Task RefreshOfficialPluginsList()
                 return;
             }
 
-            var data = JsonSerializer.Deserialize<Dictionary<string, PluginAlias>>(await response.Content.ReadAsStringAsync());
+            var data = await JsonSerializer.DeserializeAsync(await response.Content.ReadAsStreamAsync(), JsonGenerated.Default.DictionaryStringPluginAlias);
```

Comment:
This can be simplified with an extension method from `System.Net.Http.Json`
```suggestion
            var data = await response.Content.ReadFromJsonAsync(JsonGenerated.Default.DictionaryStringPluginAlias);
```